### PR TITLE
The impact of signature forgery attacks

### PR DESCRIPTION
**Pull request by nimia**

### CODE REVIEW

File: draft-ietf-tls-subcerts.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Delegated Credentials for TLS
 abbrev:
-docname: draft-ietf-tls-subcerts-latest
```
Comment (grittygrease) [inline draft-ietf-tls-subcerts.md]: Please revert this change, it needs to be -latest for the tooling.

File: draft-ietf-tls-subcerts.md [hunk 2]
```diff
@@ -545,6 +550,30 @@ clients, such as browsers in incognito mode, who do not trust the service might
...
+If possible, server operators may use DC-enabled certificates only for signing
+credentials, and not for serving non-DC TLS traffic.
+Furthermore, server operators may use elliptic curve certificates for DC-enabled
+traffic, while using RSA certificates without the DelegationUsage certificate
+extension for non-DC traffic; this completely prevents such attacks.
+
+
 # Acknowledgements
 
 Thanks to David Benjamin, Christopher Patton, Kyle Nekritz, Anirudh Ramachandran, Benjamin
```
Comment (grittygrease) [inline draft-ietf-tls-subcerts.md]: Feel free to add your names here.

File: draft-ietf-tls-subcerts.md [hunk 3]
```diff
@@ -545,6 +550,30 @@ clients, such as browsers in incognito mode, who do not trust the service might
...
 
+## The impact of signature forgery attacks
+
+When TLS 1.2 servers support RSA key exchange, they may be vulnerable to attacks
+that allow forging an RSA signature over an arbitrary message [BLEI].
+{{?RFC5246}} (Section 7.4.7.1.) describes a mitigation strategy requiring
+careful implementation for thwarting these attacks.
+Experience shows that in practice, server implementations may fail to fully
+thwary these attacks due to the complexity of this mitigation
+(TODO, NA: Should we cite the ROBOT paper here?).
```
Comment (grittygrease) [inline draft-ietf-tls-subcerts.md]: Yes, the reference is warranted.

File: draft-ietf-tls-subcerts.md (hunk 1 above)
Comment (nimia) [reply draft-ietf-tls-subcerts.md]: Done.

File: draft-ietf-tls-subcerts.md (hunk 2 above)
Comment (nimia) [reply draft-ietf-tls-subcerts.md]: Thanks! Done.

File: draft-ietf-tls-subcerts.md (hunk 3 above)
Comment (nimia) [reply draft-ietf-tls-subcerts.md]: Thanks, done.
